test/ParseableInterface/stored-properties.swift

```diff
@@ -52,6 +52,15 @@ public struct HasStoredProperties {
   // CHECK: private var _: [[BOOL]]
   private var privateVar: Bool
 
+  // CHECK: @_hasStorage @_hasInitialValue public var storedWithObserversInitialValue: [[INT]] {
+  // RESILIENT: {{^}}  @_hasInitialValue public var storedWithObserversInitialValue: [[INT]] {
```

Comment:
That `_hasInitialValue` is suspicious. We shouldn't present that knowledge in a resilient struct. (And arguably not in a fragile struct either, but let's not worry about that right now.)

Reply:
That makes sense -- that's the existing behavior, so I'll fix that in a follow-up patch.